molsysmt/item/has_attribute.py

```diff
@@ -1,25 +0,0 @@
-from molsysmt._private.lists_and_tuples import is_list_or_tuple
```

Comment:
Let's discuss if this file should be remove now. 👍🏻

Reply:
File is repeated in _basic/has_attribute.py_

Reply:
OK. You are right.